### PR TITLE
Setup and test clean-up

### PR DESCRIPTION
**Pull request by bsipocz**
This PR does a few cleanup steps:

 * removes astropy dev install hack
 * makes sure that the egg_info builds properly
 * bumps the version to 0.2.dev as v0.1 is out
 * remove the unused parts of setup.py and travis.yml
 * adds numpy as a dependency (though this is totally unnecessary as having astropy as dependency ensures that numpy is a dependency, too)

Tests are still failig, the reason is that a lot has happened in astropy dev compared to https://github.com/astropy/astropy/pull/5241, e.g more builds passes when that branch was specified:
https://travis-ci.org/bsipocz/saba/builds/179460199

### CODE REVIEW

**Comment by bsipocz** [comment]
@nocturnalastro - This depends on the upstream astropy issue so please don't merge until that one is merged and I remove the workaround (atm it still has some real failures).

In the meantime I have two questions:
 * Is there a reason why you commented out the docs build on travis? If no particular reason, I'll add that back, too. 
 * could you switch off the coveralls bot to post for PRs?

Also with this clean up PR we can cut back the CI time by ~50%, I'm sure @hamogu will appreciate that :)

**Comment by nocturnalastro** [comment]
@bsipocz - It's probably commented out because we didn't have sphinx docs when I started implementing this and then I forgot about it. 
And coveralls comments are now off.

**Comment by bsipocz** [comment]
I've removed the pointer to my astropy PR, so once that one is merge, travis needs to be restarted here and should pass.

**Comment by bsipocz** [comment]
Also, I wonder why the ipython code blocks were used instead of a normal python code blocks in the docs? Sure, it can be installed, but I think is totally unnecessary dependency.

**Comment by nocturnalastro** [comment]
I agree it's not needed.

**Comment by bsipocz** [comment]
Anyway, I leave it now, it can always be fixed at some point in the future.

**Comment by bsipocz** [comment]
closed & reopened to retrigger travis

File: docs/conf.py [hunk 1]
```diff
@@ -240,12 +234,12 @@
 
 # The name of an image file (relative to this directory) to place at the top
 # of the sidebar.
-html_logo ="saba_logo.png"
+html_favicon = os.path.join('_static', 'saba_logo.png')
```
Comment (nocturnalastro) [inline docs/conf.py]: shouldn't this still be `html_logo`

Comment (bsipocz) [reply docs/conf.py]: yep, copied from the other line but forgot to update. Done now.

**Comment by nocturnalastro** [comment]
@bsipocz thanks for all the improvements :) 
Are you happy for me to merge this?

**Comment by bsipocz** [comment]
yes, sure go ahead. Also if you have spare time to make a 0.1.1 release, this would probably solve the issues we saw for the conda packaging.

**Comment by nocturnalastro** [comment]
Sure I'll do it now :).

**Comment by nocturnalastro** [comment]
@bsipocz The new version should be up on pypi now :)

**Comment by bsipocz** [comment]
@nocturnalastro - It seems that the source code wasn't uploaded to pypi, there is only an empty entry for the new release: https://pypi.python.org/pypi/saba/0.1.1

**Comment by nocturnalastro** [comment]
@bsipocz  it seems to have a problem with 0.1.1 so i put an "a" after and it uploaded fine.

**Comment by nocturnalastro** [comment]
@bsipocz The readthedocs build is failing. As `astropy-helpers` is missing.
I've been playing around, adding `astropy-helpers` to pip-requirements both the one in docs and root. 
But the only thing which seems to fix the build is adding `astropy-helpers` back into `.rtd-environment.yml `
Am I missing something?

**Pull request merged**